src/data/cel.rs

```diff
@@ -273,46 +362,39 @@ impl Clone for Attribute {
 }
 
 impl Attribute {
-    pub fn get(&self) -> Value {
+    pub fn get(&self) -> Result<Value, PropertyError> {
         match &self.cel_type {
             Some(t) => match t {
-                ValueType::String => get_attribute::<String>(&self.path)
-                    .expect("Failed getting to known attribute")
+                ValueType::String => Ok(get_attribute::<String>(&self.path)?
                     .map(|v| Value::String(v.into()))
-                    .unwrap_or(Value::Null),
-                ValueType::Int => get_attribute::<i64>(&self.path)
-                    .expect("Failed getting to known attribute")
+                    .unwrap_or(Value::Null)),
```

Comment:
I think this was wrong all along. if `self.cel_type` is e.g. `String`, we _expect_ a `String`, and in cel this can't be `Null`... now, I wonder if we should add a `self.nullable: bool` and use that to either fallback to `Null`, as here, or just return `PropertyError` from here?